bleak/backends/dotnet/client.py

```diff
@@ -141,7 +137,7 @@ def _ConnectionStatusChanged_Handler(sender, args):
         self._requester.ConnectionStatusChanged += _ConnectionStatusChanged_Handler
 
         # Obtain services, which also leads to connection being established.
-        services = await self.get_services()
+        _ = await self.get_services()
```

Comment:
why do we event want the result of the function if we are not going to use it anywhere
Suggestion is to just leave " await self.get_services()" on this row